docker-compose/db/postgres-setup.sql

```diff
@@ -0,0 +1 @@
+CREATE DATABASE identityvault;
```

Comment:
The settings.yaml file has the default database name as "identity-vault". Let's keep the name consistent.